packages/fxa-content-server/app/scripts/templates/sign_up_password.mustache

```diff
@@ -43,17 +43,20 @@
 
       {{^isCWTSOnSignupPasswordEnabled}}
       <section class="get-involved">
-        {{#isAnyNewsletterEnabled}}
-        <header>
-          <h2>{{#t}}Practical knowledge is coming to your inbox. Sign up for more:{{/t}}</h2>
-        </header>
-        {{/isAnyNewsletterEnabled}}
+      {{^isInPocketMigration}}
+          {{#isAnyNewsletterEnabled}}
+              <header>
+                  <h2>{{#t}}Practical knowledge is coming to your inbox. Sign up for more:{{/t}}</h2>
+              </header>
+          {{/isAnyNewsletterEnabled}}
 
-        {{#newsletters}}
-          <div class="input-row marketing-email-optin-row">
-            <input id="{{slug}}" type="checkbox" class="marketing-email-optin" value="{{slug}}"><label for="{{slug}}">{{label}}</label>
-          </div>
-        {{/newsletters}}
+          {{#newsletters}}
+              <div class="input-row marketing-email-optin-row">
+                  <input id="{{slug}}" type="checkbox" class="marketing-email-optin" value="{{slug}}"><label
+                      for="{{slug}}">{{label}}</label>
```

Comment:
seems like these could be 2 space indents, and maybe this tag belongs on the line above?